### PR TITLE
Addressed text wrap property of mega menu in mobile view

### LINKED ISSUE #188: The text-wrap property of the mega menu causes an undesirable flicker when expanding collapsing.
PR from weaver-component:
`https://github.com/TAMULib/weaver-components/pull/217`

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #188

### CODE REVIEW

File: src/app/tl-header/tl-header.component.scss [hunk 1]
```diff
@@ -403,6 +403,23 @@
     left: 0;
     z-index: 1100;
     transition: width 0.5s ease-in-out, opacity 0.5s ease-in-out;
+    .flex-fill.align-self-stretch {
```
Comment (jeremythuff) [inline src/app/tl-header/tl-header.component.scss:406]: Please target identity based classes instead of functional classes.

**Review by jeremythuff — CHANGES_REQUESTED** [review]

File: src/app/tl-header/tl-header.component.scss (hunk 1 above)
Comment (rmathew1011) [reply src/app/tl-header/tl-header.component.scss]: Removed the boostrap based functional classes and added the div section instead.

File: src/app/tl-header/tl-header.component.scss [hunk 2]
```diff
@@ -403,6 +403,22 @@
     left: 0;
     z-index: 1100;
     transition: width 0.5s ease-in-out, opacity 0.5s ease-in-out;
+    div {
```
Comment (jeremythuff) [inline src/app/tl-header/tl-header.component.scss:406]: Let's add a class to the wvr component and target that.

**Review by jeremythuff — CHANGES_REQUESTED** [review]

File: src/app/tl-header/tl-header.component.scss (hunk 2 above)
Comment (rmathew1011) [reply src/app/tl-header/tl-header.component.scss:406]: mobile-menu-content class added to wvr header component and that is targeted.

**Pull request closed without merging**